pkgs/servers/routinator/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, fetchFromGitHub, rustPlatform, rsync }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "routinator";
+  version = "0.4.0";
+
+  src = fetchFromGitHub {
+    owner = "NLnetLabs";
+    repo = "routinator";
+    rev = "${version}";
```

Comment:
Sorry missed this first time around...
`rev = version;`

Reply:
@0x4A6F Sorry, my bad here. Should actually be `rev = "v${version}";`.

Reply:
@aanderse Now fixed.